test/IXMPTestApp/MSTest/IXMPTestApp.csproj

```diff
@@ -17,10 +17,12 @@
       <Version>1.0.47</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Net.Native.Compiler">
-      <Version>2.2.1</Version>
+      <Version>2.2.3</Version>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>5.2.2</Version>
+      <Version>6.2.8</Version>
```

Comment:
There is also a project.json in this dir that should be updated.

Reply:
I'll try to delete these json files and see what happens. They _should_ be unused.